client/lib/plans/constants.js

```diff
@@ -47,6 +50,9 @@ export const plansList = {
 		getTitle: () => i18n.translate( 'Free' ),
 		getPriceTitle: () => i18n.translate( 'Free for life' ),
 		getProductId: () => 1,
+		getStoreSlug: () => PLAN_FREE,
+		getPathSlug: () => 'beginner',
```

Comment:
Just asking: I've seen this used somewhere else before. Was it a deprecated `product_name` for the Free plan?